packages/legend-graph/src/graphManager/GraphManagerState.ts

```diff
@@ -168,6 +170,37 @@ export class GraphManagerState extends BasicGraphManagerState {
   systemModel: SystemModel;
   graph: PureModel;
 
+  isPrimitiveTypeElement(
```

Comment:
Sorry, let's move these methods to `BasicGraphManagerState`. Also, let's make the following adjustments:
1. Remove `isPrimitiveTypeElement()`
2. Adjust `isSystemElement()` to include `ROOT_PACKAGE_NAME.CORE`
3. Rename `isMainElement()` to `isMainGraphElement()`

Reply:
Also, I think it's nice to track down similar usage and swap out to use these instead. You can start by making a search for places where we use `getElementRootPackage()` in the code.